recipes/BEHST/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+
+./download_behst_data.sh
```

Comment:
@daler I tried to put the executable files in `$PREFIX/bin` but my `$PREFIX` folder (which is `/opt/conda/conda-bld/behst_1498494985724`) does not contain any `/bin/` folder.

I switched to `cd $SRC_DIR/bin` and everything works fine on Travis. Is this setting correct? Thanks